gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorage.java

```diff
@@ -421,6 +421,15 @@ void compose(String bucketName, List<String> sources, String destination, String
    */
   public AtomicLong getObjectStatistics(GoogleCloudStorageStatistics key);
 
+  /**
+   * To get the value object related statistics from GCS
+   *
+   * @param key name of the http related statistics key
+   */
+  default AtomicLong getHttpStatistics(GoogleCloudStorageStatistics key) {
+    throw new UnsupportedOperationException("Implemented in GoogleCloudStorageImpl");
```

Comment:
Please change the description. Interface should not refer to impl

Reply:
Updated

Reply:
Updated